app/controllers/restaurants_controller.rb

```diff
@@ -1,9 +1,11 @@
 class RestaurantsController < ApplicationController
   def index
     @categories = RestaurantCategory.all
+    @restaurants = Restaurant.all
```

Comment:
Perhaps you should look into 'group_by' and group by RestaurantCategory?

Reply:
Also good input - we will keep it in mind for the right story. This story is just about displaying restaurants and linking a restaurant to a restaurant page.

Reply:
😎